src/screens/NewKeeperAppScreen/NewKeeperAppScreen.tsx

```diff
@@ -161,15 +160,30 @@ function NewKeeperApp({ navigation }: { navigation }) {
       </Box>
     );
   }
-
+  function HeaderView({ title, subtitle }) {
+    return (
+      <Box>
+        <Text
+          numberOfLines={1}
+          style={[styles.addWalletText, { fontSize: 16 }]}
+          color={`${colorMode}.headerText`}
+          testID={`text_create_header_title`}
+        >{title}</Text>
+        <Text
+          style={[styles.addWalletDescription]}
+          color={`${colorMode}.black`}
+          testID={`text_ceate_header_subtitle`}
+        >{subtitle}</Text>
+      </Box>
+    )
+  }
   return (
     <ScreenWrapper barStyle="dark-content" backgroundcolor={`${colorMode}.primaryBackground`}>
       <Box style={{ marginTop: hp(30) }}>
         <Box style={styles.headerContainer}>
-          <KeeperHeader
+          <HeaderView
```

Comment:
Why was this component replaced?

Reply:
In that component, There is extra horizontal padding. The new UX does not need extra padding

Reply:
add optional custom padding as props to the same component maybe?

Reply:
Added.